src/app/COMPONENTS/task/task.component.html

```diff
@@ -0,0 +1,24 @@
+<app-create-task />
+
+<ul *ngIf="tasks$ | async as tasks">
```

Comment:
no reason to async the tasks twice here, you can async it the 1st time then use tasks variable in *ngFor